### DISCUSSION TITLE
REopt connection issue

### DISCUSSION THREAD

**Discussion by Mo11068** [urbanopt/urbanopt-cli#479] [Q&A]
Hi, 

Trying to process results using `uo process -r ` and getting `REopt connection Failed (RuntimeError)`. I have followed the API key instructions and addition to the env. When I am looking into `URBANoptCLI_0.13.0/gems/ruby/2.7.0/gems/urbanopt-reopt-0.12.0/lib/urbanopt/reopt/reopt_lite_api.rb` and running the API calls on the browser using the following, I still see the connection issue which does not seem to relate to my API <img width="490" alt="Screenshot 2024-09-13 at 9 18 29 AM" src="https://github.com/user-attachments/assets/297972db-d053-4d95-8259-278bd0596652">. 

> @uri_submit = URI.parse("https://developer.nrel.gov/api/reopt/v3/job?api_key=#{@nrel_developer_key}")
> @uri_submit_outagesimjob = URI.parse("https://developer.nrel.gov/api/reopt/v3/outagesimjob?api_key=#{@nrel_developer_key}")

Would appreciate any help! 
<img width="1497" alt="Screenshot 2024-09-13 at 9 19 29 AM" src="https://github.com/user-attachments/assets/9f1f8af4-7bf8-4da4-814e-bdeab280d127">

**Comment by kflemin** [comment]
We have recently released newer versions of URBANopt which addresses many prior issues. If you are still experiencing issues related to this after upgrading, please let us know.
